src/command/defaults/TimeCommand.php

```diff
@@ -72,16 +72,19 @@ public function execute(CommandSender $sender, string $commandLabel, array $args
 			return true;
 		}elseif($args[0] === "query"){
 			if(!$sender->hasPermission("pocketmine.command.time.query")){
-				$sender->sendMessage($sender->getServer()->getLanguage()->translateString(TextFormat::RED . "%commands.generic.permission"));
+				$sender->sendMessage(TextFormat::RED . "%commands.generic.permission");
 
 				return true;
 			}
+
 			if($sender instanceof Player){
 				$world = $sender->getWorld();
+				$message = $sender->getLanguage()->translateString("commands.time.query", [$world->getTime()]);
 			}else{
 				$world = $sender->getServer()->getWorldManager()->getDefaultWorld();
+				$message = $sender->getServer()->getLanguage()->translateString("commands.time.query", [$world->getTime()]);
```

Comment:
The message duplication here can also be reverted by moving the `getLanguage()` function to the CommandSender class

Reply:
If you go to extreme measures, you can pass the Language when initializing CommandSender, so that it is stored there in public access